fe/src/main/java/org/apache/doris/analysis/FunctionCallExpr.java

```diff
@@ -303,30 +300,20 @@ private void analyzeBuiltinAggFunction(Analyzer analyzer) throws AnalysisExcepti
                          "group_concat requires first parameter to be of type STRING: " + this.toSql());
             }
 
```

Comment:
Why this 2 judgement can be removed?

Reply:
Because there has the `type.isStringType()` check, HLL and bitmap is not string type. we needn't check HLL and bitmap again.

Reply:
ok